### PR TITLE
Add new question in general-cs-quiz and update modal-response

### PR DESCRIPTION
**Pull request by humsarika**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes 
issue #29

### CODE REVIEW

**Review by humsarika — COMMENTED** [review]
@jdwilkin4, I've removed two linux-quiz question from security-quiz

File: src/data/general-cs-quiz.ts [hunk 1]
```diff
@@ -1109,6 +1109,16 @@ const generalCSQuiz = [
...
+  {
+    Question: "Where is the CPU located on a computer?",
+    Answer: "The motherboard",
+    Distractor1: "Random Access Memory",
+    Distractor2: "Graphics Processing Unit",
+    Distractor3: "Solid State Drive",
+    Explanation: "The CPU is located on the motherboard of a computer.",
+    Link: "https://www.freecodecamp.org/news/what-is-cpu-meaning-definition-and-what-cpu-stands-for/",
+
+  },
```
Comment (jdwilkin4) [inline src/data/general-cs-quiz.ts:1121]: This question is already located in the IT quiz.
Please remove it here

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by humsarika** [comment]
@jdwilkin4  , I have done the requested changes.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Comment by humsarika** [comment]
@jdwilkin4 Thank you so much for merging my PR.

**Pull request merged**